Doc/reference/datamodel.rst

```diff
@@ -750,14 +753,15 @@ Modules
 
    Predefined (writable) attributes: :attr:`__name__` is the module's name;
    :attr:`__doc__` is the module's documentation string, or ``None`` if
-   unavailable; :attr:`__annotations__` (optional) is a dictionary containing
-   :term:`variable annotations <variable annotation>` collected during module
-   body execution; :attr:`__file__` is the pathname of the file from which the
+   unavailable; :attr:`__file__` is the pathname of the file from which the
    module was loaded, if it was loaded from a file. The :attr:`__file__`
    attribute may be missing for certain types of modules, such as C modules
    that are statically linked into the interpreter; for extension modules
    loaded dynamically from a shared library, it is the pathname of the shared
-   library file.
+   library file; :attr:`__annotations__` (optional) is a dictionary containing
```

Comment:
This might work better as a bulleted list, instead of a long string of semicolon-separated fragments.

Reply:
Though I take neither credit nor blame for the original formatting, and such improvements are verging on mission creep, I'll still give it a go.  I'm going to try a "definition list" first and see if I can make that work; failing that, a bulleted list as you suggest.  Either way I'll do that in both spots, modules and classes.

Reply:
I quite like how that turned out!  I invite you to see if you agree.

It makes me wonder if I should now touch up the `Special read-only attribute:` paragraph in the **Modules** section somehow.  On the other hand, there are a bunch of previous instances of *special read-only attributes* sprinkled through the preceding text, so it's not completely out of line.